en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -1,7 +1,62 @@
 # Silverstripe CMS 5.0.0-beta1
```

Comment:
I'm hoping that the "unreleased" timeframe will be small enough that this doesn't matter.

Reply:
Done.

Reply:
Still not done? Please look at the actual suggestion. It needs to have:
```md
---
title: 5.0.0-beta1 (unreleased)
---

# 5.0.0-beta1 (unreleased)
```

The title must be simplified (because that's standard to literally all the other changelogs) and it needs the title block separate to the heading, because that controls what's displayed on the card in the list of changelogs.